packages/string-templates/src/utilities.ts

```diff
@@ -66,3 +66,16 @@ export const btoa = (plainText: string) => {
 export const atob = (base64: string) => {
   return Buffer.from(base64, "base64").toString("utf-8")
 }
+
+export const prefixStrings = (
```

Comment:
Please add unit tests for this method!

Reply:
Generally we don't unit test individual functions anymore as a black box test is preferred - it is better to confirm this works through the usage of it than confirm an individual function works (if we started to go down the path of testing all functions we'd never finish!)

Reply:
Fair enough, it was mostly the regex that made me nervous, but if you're happy it's covered in other tests then all good!

Reply:
Regex always makes me nervous too - but all its doing is taking a user input, escaping anything which isn't regex friendly, then making a list of strings to find - the JS for this would be quite simple but regex is *a lot* faster and performance in this area is critical!